config.microsoftInternal.dsc

```diff
@@ -41,7 +41,10 @@ export const pkgs = isMicrosoftInternal ? [
     { id: "Drop.RemotableClient.Interfaces", version: "17.150.28901-buildid9382555", dependentPackageIdsToSkip: ["*"] },
     { id: "ItemStore.Shared", version: "17.150.28901-buildid9382555", dependentPackageIdsToSkip: ["*"], dependentPackageIdsToIgnore: ["BuildXL.Cache.Hashing"] },
     { id: "Microsoft.VisualStudio.Services.BlobStore.Client.Cache", version: "17.150.28901-buildid9382555", dependentPackageIdsToSkip: ["*"], dependentPackageIdsToIgnore: ["BuildXL.Cache.Hashing", "BuildXL.Cache.Interfaces", "BuildXL.Cache.Libraries", "BuildXL.library.forAzDev"] },
-    { id: "Microsoft.Windows.Debuggers.SymstoreInterop", version: "0.61" },
+    { id: "Microsoft.Windows.Debuggers.SymstoreInterop", version: "1.0.1" },
+    { id: "Symbol.App.Core", version: "17.150.28901-buildid9382555", dependentPackageIdsToSkip: ["*"], dependentPackageIdsToIgnore: ["BuildXL.Cache.Hashing", "BuildXL.Cache.Interfaces", "BuildXL.Cache.Libraries", "BuildXL.library.forAzDev"] },

```

Comment:
>"Symbol.App.Core" [](start = 10, length = 17)

Please make sure that our resulting NuGet packages are below 500M.

Reply:
They are (adding 90MB unzipped to ~340MB unzipped)

---
In reply to: [313637307](https://github.com/microsoft/BuildXL/pull/727#discussion_r313637307) [](ancestors = 313637307)